backend/dataall/modules/worksheets/services/worksheet_permissions.py

```diff
@@ -38,12 +38,17 @@
 RUN ATHENA QUERY
 """
 RUN_ATHENA_QUERY = 'RUN_ATHENA_QUERY'
+RUN_ATHENA_QUERY_TENANT = 'RUN_ATHENA_QUERY_TENANT'
 
 ENVIRONMENT_INVITED.append(RUN_ATHENA_QUERY)
 
 ENVIRONMENT_INVITATION_REQUEST.append(RUN_ATHENA_QUERY)
 
 ENVIRONMENT_ALL.append(RUN_ATHENA_QUERY)
+ENVIRONMENT_ALL.append(RUN_ATHENA_QUERY_TENANT)
 
 RESOURCES_ALL.append(RUN_ATHENA_QUERY)
 RESOURCES_ALL_WITH_DESC[RUN_ATHENA_QUERY] = 'Run Worksheet Athena queries on this environment'
+
+TENANT_ALL.append(RUN_ATHENA_QUERY_TENANT)
+TENANT_ALL_WITH_DESC[RUN_ATHENA_QUERY_TENANT] = 'Run Worksheet Athena queries on any environment'
```

Comment:
The permission `RUN_ATHENA_QUERY_TENANT` is not used for what the definition says. It is used to restrict who can download the results of a worksheet. It would be better to have a DOWNLOAD_ATHENA_QUERY_RESULTS permission

Reply:
or re-use the RUN_ATHENA_QUERY permission

Reply:
It should not be added to TENANT_ALL, instead is should be treated as any other Worksheet resource permission.  It should be defined in line 24 as `DOWNLOAD_ATHENA_QUERY_RESULTS = "DOWNLOAD_ATHENA_QUERY_RESULTS"` and then added to `WORKSHEET_ALL` and that's it for the definition

Reply:
Read the comments on the usage of this permission

Reply:
Agreed